readme.md

```diff
@@ -162,6 +165,7 @@
 
 - [prolog.vim](https://github.com/adimit/prolog.vim) - Prolog integration for Vim.
 - [ediprolog](https://github.com/triska/ediprolog) - Prolog integration for Emacs.
+- [Ciao mode for Emacs](https://github.com/ciao-lang/ciao_emacs) - Ciao (and Prolog) Emacs mode.
```

Comment:
From the [contributing guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md):

> - Additions should be added to the bottom of the relevant category.
> - Make an individual pull request for each suggestion.